tests/source/configs/comment_width/5801.rs

```diff
@@ -0,0 +1,16 @@
+// rustfmt-comment_width: 120
+// rustfmt-wrap_comments: true
+// rustfmt-unstable_features: true
```

Comment:
You should be able to omit the `ustable_features` option when writing tests.

Reply:
Ah ok 👍 I just lifted the config from the issue, let me remove that now